src/bot/interactions/message_commands/manage_products.js

```diff
@@ -139,7 +139,8 @@ module.exports = {
                             iconURL: `${client.user.displayAvatarURL({ dynamic: true })}`,
                             name: 'Inertia Lighting | Products Manager',
                         },
-                        description: `${action_to_perform === 'add' ? 'Added' : 'Removed'} ${db_roblox_product.code} ${action_to_perform === 'add' ? 'to' : 'from'} ${user_to_modify}!`,
+                        title: "Success!",
+                        description: `${action_to_perform === 'add' ? 'Added' : 'Removed'} \`\`${db_roblox_product.name}\`\` *(${db_roblox_product.code})* ${action_to_perform === 'add' ? 'to' : 'from'} ${user_to_modify}.`,
```

Comment:
```suggestion
                        description: `${action_to_perform === 'add' ? 'Added' : 'Removed'} \`${db_roblox_product.code}\` ${action_to_perform === 'add' ? 'to' : 'from'} ${user_to_modify}.`,
```
Double backtick syntax is not supported by all Discord clients.
And you shouldn't list both the product code and product name, as it is redundant.